lua/entities/gmt_instrument_base/shared.lua

```diff
@@ -64,7 +64,14 @@ function ENT:GetSound( snd )
 		return nil
 	end
 	
-	return "gmodtower/lobby/instruments/" .. self.Instruments[self:GetNWInt("CurrentInstrument")] .. "/" .. snd .. ".ogg"

```

Comment:
You can do:
return "gmodtower/lobby/instruments/" .. (self.Instruments[self:GetNWInt("CurrentInstrument")] or "Piano") .. "/" .. snd .. ".ogg"